src/ontology/mondo-ingest.Makefile

```diff
@@ -341,7 +342,7 @@ documentation: j2 $(ALL_DOCS) unmapped-terms-docs mapped-deprecated-terms-docs s
 build-mondo-ingest:
 	$(MAKE) refresh-imports exclusions-all slurp-all mappings matches \
 		mapped-deprecated-terms mapping-progress-report \
-		recreate-unmapped-components sync documentation
+		recreate-unmapped-components documentation
```

Comment:
@joeflack4 can you make an issue about this? A clean run of `make sync` does not work because we get an error: `no goal for reports/doid.subclass.direct-in-mondo-only.tsv`. There is indeed no such goal!

Reply:
@matentzn Sorry about this. But it it strange. There is a goal, and I'm getting no such error. I created an issue though: #398

Reply:
Ok, can you open a PR to reinstate the sync pipeline, try a fresh checkout of the repo (different folder) and see if you can replicate the problem?

Reply:
Alright

Reply:
@matentzn I'm just tried a fresh clone, running on `main`, before and after clearing `reports/` of any `sync` related outputs.

I'm not getting any errors, but no outputs either. Strange.
<details><summary>Example of just DOID</summary>
<p>

```
joeflack4@MacBook-Pro ontology % sh run.sh make sync-subclassof-doid
Running  with -Xmx20G of memory for ROBOT and Java-based pipeline steps.
make: Entering directory '/work/src/ontology'
...
Makefile:447: warning: ignoring old recipe for target 'components/omim.owl'
...
make reports/doid.subclass.direct-in-mondo-only.tsv
make[1]: Entering directory '/work/src/ontology'
...
mondo-ingest.Makefile:66: warning: overriding recipe for target 'components/omim.owl'
make pip-oaklib
make[2]: Entering directory '/work/src/ontology'
...
python3 -m pip install --upgrade oaklib
Defaulting to user installation because normal site-packages is not writeable
...
make[2]: Leaving directory '/work/src/ontology'
python3 ../scripts/sync_subclassof.py \
--outpath-added reports/doid.subclass.added.robot.tsv \
--outpath-added-obsolete reports/doid.subclass.added-obsolete.robot.tsv \
--outpath-confirmed reports/doid.subclass.confirmed.robot.tsv \
--outpath-direct-in-mondo-only reports/doid.subclass.direct-in-mondo-only.tsv \
--mondo-db-path tmp/mondo.db \
--mondo-ingest-db-path tmp/mondo-ingest.db \
--mondo-mappings-path tmp/mondo.sssom.tsv \
--onto-config-path metadata/doid.yml
/work/src/ontology/../scripts/sync_subclassof.py:47: DeprecationWarning:
Pyarrow will become a required dependency of pandas in the next major release of pandas (pandas 3.0),
(to allow more performant data types, such as the Arrow string type, and better interoperability with other libraries)
but was not found to be installed on your system.
If this would cause problems for you,
please provide us feedback at https://github.com/pandas-dev/pandas/issues/54466

  import pandas as pd

```

</p>
</details> 

I don't have anything in my `tmp/` folder but it's still running because this isn't a fresh ODK container either.

@matentzn What ODK version are you using? I'm using `dev`.

I'm gonna try `make` locally without ODK and see what the results are.

Reply:
Ok, immediately when I tried `make` outside of ODK I got what you got:
`make[1]: *** No rule to make target `reports/doid.subclass.direct-in-mondo-only.tsv'.  Stop.`

Reply:
@matentzn Gonna move discussion to:
- #402